client/components/KeyMetricsBanner/KeyMetricsBanner.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { Alert } from 'react-bootstrap';
+import { KEY_METRICS_QUERY } from './queries';
+import { useQuery } from '@apollo/client';
+
+const KeyMetricsBanner = () => {
+  const { data: keyMetricsQuery } = useQuery(KEY_METRICS_QUERY);
+  const {
+    date,
+    verdictsCount,
+    commandsCount,
+    contributorsCount,
+    verdictsLast90Count
+  } = keyMetricsQuery?.keyMetrics ?? {};
+  return (
+    <Alert
```

Comment:
May just want a bit of horizontal margins here but this approach looks good to me!